### ISSUE TITLE
代码错误问题

### ISSUE THREAD

**Issue by learningInnovativetigers** [apache/incubator-dubbo-website#334]
通过 API 编程进行泛化调用 标题下的第一块代码的第一行多了ApplicationConfig application = new ApplicationConfig()
参数或返回值是 POJO 的场景 标题下的消费端的调用代码中,HashMap的命名和下面调用使用的不一致.

**Comment by kexianjun** [comment]
you are welcome to send pr to fix it.

**Comment by caojiele** [comment]
> 通过 API 编程进行泛化调用 标题下的第一块代码的第一行多了ApplicationConfig application = new ApplicationConfig()
> 参数或返回值是 POJO 的场景 标题下的消费端的调用代码中,HashMap的命名和下面调用使用的不一致.

Hi, the problem is [here](http://dubbo.apache.org/en-us/docs/user/configuration/api.html) ?
//Application Info
ApplicationConfig application = new ApplicationConfig();👈

**Comment by kexianjun** [comment]
> > 通过 API 编程进行泛化调用 标题下的第一块代码的第一行多了ApplicationConfig application = new ApplicationConfig()
> > 参数或返回值是 POJO 的场景 标题下的消费端的调用代码中,HashMap的命名和下面调用使用的不一致.
> 
> Hi, the problem is [here](http://dubbo.apache.org/en-us/docs/user/configuration/api.html) ?
> //Application Info
> ApplicationConfig application = new ApplicationConfig();👈

NO, Seems in [generic-reference](http://dubbo.apache.org/zh-cn/docs/user/demos/generic-reference.html), @learningInnovativetigers I have checked it again,seems it has been fixed, can you confirm it?
